app/src/main/java/com/github/mateo762/myapplication/profile/ProfileActivity.kt

```diff
@@ -263,8 +294,41 @@ class ProfileActivity : BaseActivity(), CoroutineScope {
                     textViews.add(textView)
                     top += 10
                 }
+
+                // Set the number of habits and the average repetitions per week
+                averageRepetitionsPerWeek = daysCount.sum() / daysCount.size
+                habitCountText.text = "Posted habits: $numberOfhabits"
```

Comment:
hardcoded string, you can use it with strings.xml with parameters. You can check how it is used in the case of "create_habit_end_time_text" strings resource